velox/exec/tests/utils/TpchQueryBuilder.h

```diff
@@ -105,6 +106,11 @@ class TpchQueryBuilder {
   static const std::unordered_map<std::string, std::vector<std::string>>
       kTables_;
   static const std::vector<std::string> kTableNames_;
+
+  static constexpr char* kLineitem = (char*)"lineitem";
```

Comment:
You should not need the cast. Why do you need them?
try `static constexpr const char* kLineitem`?

Reply:
Thanks for the suggestion, cast was added to get rid of the following warning:
warning: ISO C++11 does not allow conversion from string literal to 'char *const'
Using 'constexpr const' fixed the warning as well.